index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Bowling</title>
+  <link href='public/css/style.css' rel='stylesheet' type='text/css'>
+  <script src='src/jquery-3.2.1.min.js'></script>
+  <script src='src/game.js'></script>
+  <script src='src/interface.js'></script>
+</head>
+
+<body>
+  <div id='title' class='title'><h1>Bowling!</h1></div>
+  <div id='buttons' class='buttons'>
+    <p>Enter on your roll</p>
+    <button class="rollScore"> 0 </button>
```

Comment:
You mentioned on your review request form that you struggled with the UI. I think we covered some of this in the code review on Monday morning, but I'll cover here again for reference. 

Here, rather than 10 buttons, we could use an iterator. If you have an element:

```
<div id='button-container'></div>
```

Then you could add your buttons using a for loop:

```
function buildButtons() {
    for (var index = 0; index <= 10; index++) {
        var button = $('<button class="rollScore"></button>');
        button.text(index);
        button.click(function() {
            setScore(index);
        });
        $('#button-container').append(button);
    }
}

function setScore(score) {
   .. set the score
}
```